meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mp-lpddr4-evk/lmp-base.cfg

```diff
@@ -0,0 +1,25 @@
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_SPL_FIT=y
+CONFIG_SPL_LOAD_FIT=y
+# CONFIG_SPL_FIT_IMAGE_TINY is not set
+# CONFIG_CMD_DEKBLOB is not set
+# CONFIG_SPL_DM_USB is not set
+CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_FIT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_VERBOSE=y
+CONFIG_LEGACY_IMAGE_FORMAT=y
+CONFIG_PARTITION_UUIDS=y
+CONFIG_CMD_XIMG=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+# CONFIG_ENV_IS_IN_MMC is not set
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="2:1"
+CONFIG_ENV_SIZE=0x4000
+CONFIG_CMD_IMPORTENV=y
+CONFIG_CMD_EDITENV=y
+CONFIG_CMD_SAVEENV=y
+CONFIG_USE_BOOTCOMMAND=y
+CONFIG_BOOTCOMMAND="fatload mmc ${emmc_dev}:1 ${loadaddr} /boot.scr; source ${loadaddr}; reset"
```

Comment:
We might not need to define bootcommand here as usually the board has enough to find and use a boot.scr file, but fine to keep if you want.

Reply:
Yeah most likely this was copied over from imx8mm, but we're a bit inconsistent:
```
./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mm-lpddr4-evk/lmp-base.cfg:26:CONFIG_BOOTCOMMAND="fatload mmc ${emmc_dev}:1 ${loadaddr} /boot.scr; source ${loadaddr}; reset"
./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx7ulpea-ucom/lmp-base.cfg:52:CONFIG_BOOTCOMMAND="mmc rescan; fatload mmc ${mmcdev}:1 ${loadaddr} boot.scr; source ${loadaddr}"
./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8qm-mek/lmp-base.cfg:33:CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${scriptaddr} /boot.scr; source ${scriptaddr}"
./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mq-evk/lmp-base.cfg:25:CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${loadaddr} /boot.scr; source ${loadaddr}; reset"
```

Reply:
For now, i'll keep it in and we can clean up all of the machines at once if we want to drop this.